src/Native/Runtime/portable.cpp

```diff
@@ -421,3 +421,56 @@ COOP_PINVOKE_HELPER(bool, RhpETWShouldWalkCom, ())
     ASSERT_UNCONDITIONALLY("NYI");
     return false;
 }
+
+#if !defined(_WIN32)
```

Comment:
These warts should probably go to the place where we collect temporary warts - the bootstrapper.

`platform.unix.cpp` already has a collection.